flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/operators/join/stream/StreamingMiniBatchJoinOperatorTest.java

```diff
@@ -1953,4 +1947,61 @@ public RowType getOutputType() {
                     return FlinkJoinType.FULL;
                 }
             };
+
+    private final Function<String, InternalTypeInfo<RowData>> leftTypeInfoExtractor =
+            (testDisplayName) -> {
+                if (testDisplayName.contains("HashCollision")) {
```

Comment:
Could you make this statement be consistent with the following conditional statement?

Reply:
Sorry for this stupid mistake, fixed it.